.github/workflows/gpu_benchmark.yaml

```
@@ -0,0 +1,13 @@
+name: GPU Benchmark PR
+on: [push, pull_request]
```

you might want to change this to match CI/CD improvements PR
```suggestion
on:
  push:
    branches:
      - master
  pull_request:
    paths:
      - ".github/workflows/gpu_benchmark.yaml"
      - "gpu_benchmark/**"
```